pkg/configuration/configuration.go

```diff
@@ -91,6 +92,21 @@ func IsAuthCheckEnabled(cmd *cobra.Command) bool {
 	return true
 }
 
+func CheckMinAPIVersionRestriction(cmd *cobra.Command, currentVersion int) error {
+	c := cmd
+	for c != nil {
+		if c.Annotations != nil {
+			if s, ok := c.Annotations["MinAPIVersion"]; ok {
+				if minVersion, err := strconv.ParseInt(s, 10, 64); err == nil && int64(currentVersion) < minVersion {
```

Comment:
```suggestion
				if minVersion, err := strconv.Atoi(s); err == nil && currentVersion < minVersion {
```
the api version is a predictable integer value, we could use Atoi here for brevity, to avoid typecasting.

Reply:
makes sense